pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -943,11 +943,11 @@ func (ctrl *backupDriverController) syncUploadByKey(key string) error {
 	case datamoverapi.UploadPhaseCleanupFailed:
 		newSnapshotStatusPhase = backupdriverapi.SnapshotPhaseCleanupFailed
 	default:
-		ctrl.logger.Infof("syncUploadByKey: No change needed for upload phase %s", upload.Status.Phase)
+		ctrl.logger.Debugf("syncUploadByKey: No change needed for upload phase %s", upload.Status.Phase)
 		return nil
 	}
 	snapshotStatusFields := make(map[string]interface{})
-	ctrl.logger.Infof("syncUploadByKey: calling updateSnapshotStatusPhase %s/%s", snapshot.Namespace, snapshot.Name)
+	ctrl.logger.Debugf("syncUploadByKey: calling updateSnapshotStatusPhase %s/%s", snapshot.Namespace, snapshot.Name)
```

Comment:
Please leave this line as it is. This line should be called constant times (depends on how many number of states of Snapshot CR) per CR.

Reply:
Same, updateSnapshotStatusPhase logs when it moves from one phase to another, there is no need of additional logging here.